src/utils/size.scss

```diff
@@ -1,4 +1,4 @@
-@import "../base/config.scss";
+@import "../config/config.scss";
```

Comment:
We'll need to do this for all the components too..  `@import "../config/config.scss";` needs to be included for each one so they can all be imported and composed upon individually within react.